### DISCUSSION TITLE
Secret for Custom github action

### DISCUSSION THREAD

**Discussion by dohuutamhuy** [community/community#25953] [Actions]
<p>Hi everyone! I’m new to the github community and github action. I’m building a custom github action that will download a binary file and run it on the VM. I use GPG to encrypt my binary file with a passphrase. Is there any way to store the passphrase as a secret on github so that every time user runs the action the passphrase is retrieved from the action repository not from the user repository?</p>

**Comment by weide-zhou** [comment, marked as answer]
<p>Hi <a class="mention" href="/dohuutamhuy">@dohuutamhuy</a>,</p>
<p>Glad to see you in Github Community Forum!</p>
<p>It’s not supported, repository encrypted secrets are not shared to another repo, hence the user repo cannot retrive the passphrase from the action repo.</p>
<p>Thanks</p>

**Comment by dohuutamhuy** [comment]
<p>Thank you for your answer, <a class="mention" href="/weide-zhou">@weide-zhou</a> ! :smiley:</p>
